hip_data_tools/aws/s3.py

```diff
@@ -122,7 +124,8 @@ def move_recursive_to_different_bucket(self, source_dir, destination_bucket_name
         for obj in source_bucket.objects.filter(Prefix=source_dir):
             if not obj.key.endswith(file_suffix_filter):
                 new_key = "{destination_dir_without_bucket_name}/{destination_file_name}".format(
-                    destination_dir_without_bucket_name=destination_dir.replace(destination_bucket_name + '/', ''),
+                    destination_dir_without_bucket_name=destination_dir.replace(
+                        destination_bucket_name + '/', ''),
                     destination_file_name=obj.key.split('/')[-1])
                 log.info("Moving s3 object from : \n%s \nto: \n%s", obj.key, new_key)
```

Comment:
this might a debug level flag, in the case of lots of files this could be a large log

Reply:
Yeah I like the large log, leaves a nice audit trail when stuff breaks in production rather than having to rerun to diagnose.